firefox-ios/Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -49,10 +50,13 @@ class LaunchCoordinator: BaseCoordinator,
     // MARK: - Terms of Service
     private func presentTermsOfService(with manager: TermsOfServiceManager,
                                        isFullScreen: Bool) {
-        let viewController = TermsOfServiceViewController(windowUUID: windowUUID)
+        let viewController = TermsOfServiceViewController(profile: profile, windowUUID: windowUUID)
         viewController.didFinishFlow = { [weak self] in
             manager.setAccepted()
             guard let self = self else { return }
+            let sendCrashReports = profile.prefs.boolForKey(AppConstants.prefSendCrashReports) ?? true
```

Comment:
Same here, curious why we have it default true instead of false 🤔

Reply:
This is a product request. Option should be true by default. :)